README.md

```diff
@@ -31,6 +31,66 @@ And below are the lower-level packages, used internally, or by advanced users.
 | [`@parity/contracts`](/packages/contracts) | [![npm (scoped)](https://img.shields.io/npm/v/@parity/contracts.svg)](https://www.npmjs.com/package/@parity/contracts) | [![README](https://img.shields.io/badge/docs-README-green.svg)](/packages/contracts#readme)      | Parity's [contracts](https://github.com/parity-contracts) as ES6 classes. |
 | [`@parity/electron`](/packages/electron)   | [![npm (scoped)](https://img.shields.io/npm/v/@parity/electron.svg)](https://www.npmjs.com/package/@parity/electron)   | [![README](https://img.shields.io/badge/docs-README-green.svg)](/packages/electron#readme)       | Control the Parity Ethereum node from Electron.                           |
 
+### Contributing
+
+#### Dependencies
+
+Install at least `yarn` version 1.4.2 and [Node.js >=10.11.0](https://nodejs.org/en/)
+
+```
+yarn --version // Should be at least 1.4.2
+```
+
+#### Tests
+
+```
+export NPM_TOKEN=''; yarn test
+```
+
+#### Build
+
+```
+export NPM_TOKEN=''; yarn build
+```
+
+#### Maintenance
+
+1. Fork the repo
+
+2. Clone your fork
+```
+git clone https://github.com/<INSERT_YOUR_GITHUB_USERNAME>/js-libs
+```
+
+3. Check outdated dependencies
+
+```
+npm outdated
+```
+
+4. Create a branch and update any dependencies that it indicates are out of date.
```

Comment:
I'd prefer that it'd be us who update dependencies. They sometimes make things break. Or even better: implement #28.